### PR TITLE
Fix css bugs and take into account top and bottom bars in phone browser

### PR DESCRIPTION
**Pull request by karenfrilya97**
#109

### CODE REVIEW

**Comment by karenfrilya97** [comment]
#107 add logout button

File: src/client/buttons/DislikeButton/DislikeButton.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import IconButton from '../IconButton';
+
+const DislikeButton = (props) => {
+  const { className, onClick } = props;
+
+  return (
+    <IconButton
+      icon=""
+      className={`dislike-button ${className}`}
```
Comment (tanrenyi) [inline src/client/buttons/DislikeButton/DislikeButton.jsx]: Usually we don't pass in `classNames` as props, but i can see why you did. (i guess you're trying to make the component reusable while at the same time you need it to be specific based on where it is rendered) 

One way i'd do it is pass in a `parent` prop to indicate where `DislikeButton` is being rendered from. Then within `DislikeButton` component, set up a `switch` statement to add the associated className.

```js
// if being rendered from Swipe component
...
<DislikeButton parent="swipe" onClick={onClick} />
...
```

```js
// in DislikeButton 
const {parent} = props;
let className="dislike-button ";
switch(parent) {
   case 'swipe': 
     className += "swipe ";
     break;
   case 'another-component':
     className += "another-className ";
     break;
 ...
```

but it's okay if you want to stick to current method. it's fine too.

File: src/client/buttons/HomeButton/HomeButton.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import IconButton from '../IconButton';
+
+const HomeButton = (props) => {
+  const { className, onClick } = props;
+  return (
+    <IconButton
+      icon=""
```
Comment (tanrenyi) [inline src/client/buttons/HomeButton/HomeButton.jsx]: is there a reason why `icon=""`? 
is this required from the doc or is it displaying wrongly for me?

File: src/client/components/Swipe/InfoPanel/InfoPanel.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        <div className="info-content">
+          <button className="info-button-close" onClick={() => handleClick(false)}>
+            <span className="text">tap here to close</span>
+            <span className="icon"></span>
+          </button>
+          <div  className="info-title">
+            { place.name + ", " + place.price }
+          </div>
+          <div  className="info-intro">
+            { place.desc }
```
Comment (tanrenyi) [inline src/client/components/Swipe/InfoPanel/InfoPanel.jsx]: image

there's something weird here. the text is cut. i can't look into it right now, but give it a shot. if you still can't fix it, let me know, we can look into it in more details!

File: src/client/components/Swipe/Swipe.jsx [hunk 4]
```diff
@@ -216,68 +215,30 @@ class Swipe extends Component {
...
-          />
+          </div>
+          <div className="swipe-header-secondary">
+            <HomeButton
+              onClick={() => this.routeChange(PATHS.trips())}
+            />
+            {places.length > 0 && (
+              <div>
+                {this.renderList(this.state.swipeList)}
+              </div>
```
Comment (tanrenyi) [inline src/client/components/Swipe/Swipe.jsx]: don't render a div outside of a function, put it within the render function and give it a `className`

so...

```js
{places.length > 0 && this.renderList(this.state.swipeList)}
```

```js
renderList(swipeList) {
   ...
   return (
   <div className="toggle-button-group">
      <ToggleButtonGroup className="toggle-buttons" name="toggle-button" value={swipeList} onChange={this.listChange}>
          <ToggleButton className="toggle-button-selected" name="food" value={1}>food</ToggleButton>
          <ToggleButton className="toggle-button-unselected" name="places" value={2}>places</ToggleButton>
       </ToggleButtonGroup>
   </div>
}
```

**Comment by karenfrilya97** [comment]
Thanks for the review! Can you check
https://bucketlist-app.slack.com/archives/CNUV3JQ9E/p1571502506000100 about
the cut line?
Thanks!

File: src/client/buttons/HomeButton/HomeButton.jsx (hunk 2 above)
Comment (renyitan) [reply src/client/buttons/HomeButton/HomeButton.jsx]: Yeap just checked it. It’s an emoji, but it was displaying wrong on my browser. So dw about it

**Pull request merged**